cmd/fleetctl/get_test.go

```diff
@@ -360,7 +360,7 @@ func TestGetHosts(t *testing.T) {
 		}, nil
 	}
 
-	ds.GetHostLockWipeStatusFunc = func(ctx context.Context, hostID uint, fleetPlatform string) (*fleet.HostLockWipeStatus, error) {
+	ds.GetHostLockWipeStatusFunc = func(ctx context.Context, host *fleet.Host) (*fleet.HostLockWipeStatus, error) {
```

Comment:
Heads-up that I refactored the `GetHostLockWipeStatus` datastore method so that it can safely validate that the MDM command results it gets are for the right host (via its host uuid). In case you see weird build errors in code that isn't yet part of the feature branch, that may be due to this signature change!